### PR TITLE
test(e2e): exam show tests

### PR DESCRIPTION
**Pull request by dzmdre**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

 [#51705](https://github.com/freeCodeCamp/freeCodeCamp/issues/51705)

### CODE REVIEW

**Comment by dzmdre** [comment]
Looks like I have no "pnpm run seed:exams" on ci and can not click on the Start exam button
https://discord.com/channels/692816967895220344/715074489422970962/1159520223645409321

**Comment by dzmdre** [comment]
Blocked by #51841

**Comment by dzmdre** [comment]
Please  review

File: .github/workflows/e2e-playwright.yml
Comment (naomi-lgbt) [inline .github/workflows/e2e-playwright.yml:1]: We should probably extrapolate this change to its own PR.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

File: e2e/exam-started-show.spec.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+      await expect(prevQuestionBtn).toBeVisible();
+      if (i != 1) {
+        await expect(prevQuestionBtn).toBeEnabled();
+      } else {
+        await expect(prevQuestionBtn).not.toBeEnabled();
+      }
+      for (let j = 0; j < 5; j++) {
+        // 4 answers
+        await page.getByTestId('exam-answer-label-' + String(j)).check();
+      }
```
[the remark this answers is not in the document]
Comment (dzmdre) [reply e2e/exam-started-show.spec.ts]: @huyenltnguyen thank you for your  review points. Just one more question from my side: should  we check  (test if its clickable) every radio button on every page?

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

**Pull request merged**